### PR TITLE
added default lists.

### PR DESCRIPTION
**Pull request by LevoraShelby**
Merging into `amazon-s3-image-upload`. Creates a shopping list and wishlist when a user signs up.

### CODE REVIEW

File: server/routes/user.js [hunk 1]
```diff
@@ -52,11 +54,26 @@ router.post(
...
+      title: "Shopping",
+      imageUrl: "https://image.flaticon.com/icons/png/512/1600/1600225.png",
+      transaction,
+    });
+    await UserListModel.create({
+      user_id: newUser.id,
+      title: "Wishlist",
+      imageUrl: "https://image.flaticon.com/icons/png/512/1600/1600225.png",
+      transaction,
+    });
```
Comment (sina-jamshidi) [inline server/routes/user.js]: actually in this case I think it's okay if we don't link creating the default lists to user creation. If making the new lists fails we wouldn't want a user to be unable to sign up

**Review by sina-jamshidi — APPROVED** [review]
Looks good but why are we merging into `amazom-s3-image-upload`?

**Comment by LevoraShelby** [comment]
It's the branch I branched off from since it has the version of UserLists I think we'll be keeping and we already have a PR up for merging that branch into dev.

File: server/routes/user.js (hunk 1 above)
Comment (LevoraShelby) [reply server/routes/user.js]: Alright. I'll make it a separate transaction tomorrow. It's late for me now so I'm going to head off to bed.

**Pull request merged**